apps/frontend/src/hooks/useAssetBalance.ts

```diff
@@ -58,7 +62,12 @@ export const useAssetBalance = (
         account,
       ]);
 
-      sub = observeCall(hashedArgs).subscribe(e => setState(e.result));
+      sub = observeCall(hashedArgs).subscribe(e =>
+        setState({
+          ...e.result,
+          parsedValue: fromWei(e.result.value, tokenDetails.decimalPrecision),
```

Comment:
I am getting these errors in the console, they are probably caused by this.
<img width="1040" alt="image" src="https://user-images.githubusercontent.com/14878713/220376704-298c4f08-b91c-42be-ae43-da30abdba37c.png">